NYTPhotoViewer/NYTInterstitialViewController.m

```diff
@@ -37,17 +38,28 @@ - (instancetype)initWithCoder:(NSCoder *)aDecoder {
     return self;
 }
 
+- (void)prepareLayout {
+    if (self.interstitialView.translatesAutoresizingMaskIntoConstraints) {
+        self.interstitialView.center = CGPointMake(CGRectGetMidX(self.view.bounds), CGRectGetMidY(self.view.bounds));
+    } else if (self.constraints == NULL) {
```

Comment:
nit: `self.constraints == nil` or `!self.constraints`. `NULL` is a `typedef` of `void *` (C) whereas `nil` is a typedef of `id` (Obj-C).